src/views/shared/TextBox.js

```diff
@@ -52,8 +52,8 @@ function ($, Handlebars, BrowserFeatures, TextBox) {
           <span class="icon input-icon {{params.icon}}"></span>\
         {{/if}}\
       {{/if}}\
-      <input type="{{type}}" placeholder="{{placeholder}}" name="{{name}}" \
-        id="{{inputId}}" value="{{value}}" autocomplete="off"/>\
+      <input type="{{type}}" placeholder="{{placeholder}}" aria-label="{{placeholder}}"\
```

Comment:
Since this is a visible element. shouldn't we be using the `aria-labelledby` attribute?
(just looked at this [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute))